spec/hangman_rules_spec.rb

```diff
@@ -0,0 +1,61 @@
+require 'spec_helper'
+require_relative '../lib/hangman_rules'
+
+RSpec.describe HangmanRules do
+
+  let(:words) {Words.new}
+
+  it "returns a random word" do
+    hangman_rules = new_hangman_rules(WordsStub.new)
+
+    word = hangman_rules.random_word()
+
+    expect(word).to eq("hello")
+  end
+
+  it "stores letters correctly guessed" do
+    hangman_rules = new_hangman_rules(words)
+
+    hangman_rules.keep_track_of_guesses("hello", "h")
```

Comment:
Hmmm... `"hello"` here feels odd. I was kinda expecting that secret word to already be kept somewhere in the bowels of `hangman_rules`?

something like 
```
hangman_rules = new HangManRules(wordsStub)
hangman_rules.guess("h")
hangman_rules.guess("e)

expect(hangman_rules.correct_guesses).to eq(["h", "e"])
expect(hangman_rules.remaining_guesses).to eq(1_000_000)
```

Reply:
You're right!! My app is missing a `guess` method!!!